### DISCUSSION TITLE
How NOT to copy the plot file from the TEMP directory to the destination?

### DISCUSSION THREAD

**Discussion by anim72** [Chia-Network/chia-blockchain#2729] [General]
Is there a way to just not do any copying after the plot.tmp file is created in the TEMP folder?

If I do not provide the -d parameter, then the plot file is copied to the folder from where the chia executable is run. If I provide a folder, then a full copy is done instead of moving the file. 

How can I omit this last step of copying the plot file to a destination directory?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
